src/main/target/STM32_UNIFIED/target.h

```diff
@@ -78,6 +78,29 @@
 #define TARGET_IO_PORTD 0xffff
 #define TARGET_IO_PORTE 0xffff
 
+#elif defined(STM32F411SX1280)
+#define TARGET_BOARD_IDENTIFIER "S4SX"
```

Comment:
`S41S` or `S41X`

Reply:
Not sure if any standard for the definition of these four character identifiers exists.